README.md

```diff
@@ -27,17 +27,17 @@ Ruby: 1.9.3, 2.X, Rails: >= 4.0. Works in pair with **Devise**, **Authlogic**, *
 
 ## Installation
 
- 1. List  Add to your Gemfile:
+ 1. List  Add to your Gemfile to the `:development` group:
 
-        gem 'any_login'
+        gem 'any_login', group: :development
 
  2. Execute in console:
 
         bundle install
 
  3. In application layout (for example `app/views/layouts/application.html.erb`) add in the bottom of the page:
 
-        <%= any_login_here %>
+        <%= any_login_here if Rails.env.development? %>
```

Comment:
Many thanks for the feedback. 

I prefer the conservative approach of the default install instructions limiting a gem with this kind of power to development environments and if developers want to do something more advanced (like enable in non-dev environments) relying on them knowing enough how to achieve that or provide an advanced instructions section in the README.

However I understand and I respect your opinion. I'll remove the README changes shortly so its just the controller changes in this PR.